<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeadersDialog</class>
 <widget class="QDialog" name="HeadersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit headers</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:app.ico</normaloff>:app.ico</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="styleSheet">
      <string notr="true">QTreeWidget::item:!selected{
    height: 25px;
	border-bottom: 1px dotted grey;
}</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionInsertAbove">
   <property name="text">
    <string>Insert above</string>
   </property>
   <property name="toolTip">
    <string>Insert new item above</string>
   </property>
  </action>
  <action name="actionInsertBelow">
   <property name="text">
    <string>Insert below</string>
   </property>
   <property name="toolTip">
    <string>Insert new item below</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove the selected item</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="text">
    <string>Insert</string>
   </property>
   <property name="toolTip">
    <string>Insert new item</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="text">
    <string>Move up</string>
   </property>
   <property name="toolTip">
    <string>Move the item up</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="text">
    <string>Move down</string>
   </property>
   <property name="toolTip">
    <string>Move the item down</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HeadersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HeadersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
